chia/rpc/wallet_rpc_api.py

```diff
@@ -1380,21 +1383,22 @@ async def cat_spend(self, request, hold_lock=True) -> EndpointResult:
             exclude_coin_amounts = [uint64(a) for a in exclude_coin_amounts]
         exclude_coin_ids: Optional[List] = request.get("exclude_coin_ids")
         if exclude_coin_ids is not None:
-            exclude_coins: Set[Coin] = {
+            exclude_coins: Optional[Set[Coin]] = {
```

Comment:
you change this type, but I don't see you changing the signature for `generate_signed_transaction()` (which this variable is passed into). Do you understand why mypy doesn't issue a warning?

Reply:
I see, it took an optional all along